.github/workflows/scan.yaml

```
@@ -5,6 +5,10 @@ on:
         required: true
       GITHUBACTIONSTOKEN:
         required: true
+    git:
+      submodules:
+        default: false
```

I think this would have to be different, per https://docs.github.com/en/enterprise-cloud@latest/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_callinputs

```suggestion
    inputs:
      submodules:
        required: false
```

(See a change below that would also have to be made)